### ISSUE TITLE
Clarification on MIMIC-IV version, preprocessing steps, and loss weight hyperparameters in DrFuse

### ISSUE THREAD

**Issue by Miiia12** [dorothy-yao/drfuse#11]
Dear authors,

Thank you for sharing your excellent work on DrFuse.
I am currently reproducing your experiments using MIMIC-IV v2.2, and I would like to clarify several details regarding the dataset preprocessing and hyperparameter settings.

After completing the preprocessing pipeline following the repository’s instructions, I obtained the following EHR dataset statistics:
Train: 41042  
Val: 4533  
Test: 11313  
Total ICU stays with EHR: 56888
However, the paper reports about 59,344 ICU stays after filtering, which is slightly higher than my result.

Could you please clarify the following points:

1.Which version of MIMIC-IV was used in your experiments — v1.0, v2.0, or v2.2?

2.Did you fully execute all filtering steps in the preprocessing scripts, such as:
REMOVE ICU TRANSFERS
REMOVE MULTIPLE STAYS PER ADMIT
REMOVE PATIENTS AGE < 18
Or were any of these steps skipped or adjusted?

3.Regarding the loss weighting hyperparameters:
lambda_attn_aux,
lambda_pred_ehr,
lambda_pred_cxr, and
lambda_pred_shared.
The code sets them to 1.0 by default, but the paper mentions 0.5. Could you please confirm the actual values used in your final reported experiments?

Thank you very much for your time and clarification!
